numpy/lib/tests/test_function_base.py

```diff
@@ -1505,6 +1506,15 @@ def test_0d(self):
         assert_(y.ndim == 0)
         assert_(y == 0)
 
+        x = 5
+        y = piecewise(x, [[True], [False]], [1, 0])
```

Comment:
This doesn't make any sense. If `x` is 0d, the conditions should be 0d too, as `[True, False]`